projects/packages/forms/src/dashboard/inbox/dataviews/index.js

```diff
@@ -277,6 +302,10 @@ export default function InboxView() {
 		return _actions;
 	}, [ isMobile, onChangeSelection, selection ] );
 
+	const resetPage = useCallback( () => {
+		view.page = 1;
+	}, [ view ] );
```

Comment:
`view` here is an element of state created by `useState()` and returned from the `useView()` hook, along with a state setter `setView()`. Is there a reason we mutate that directly here vs setting via the value with setView like this?

```
const resetPage = useCallback(() => {
  setView(prev => ({ ...prev, page: 1 }));
}, [ setView ]);
```

Reply:
Using setView causes a race condition. The page is updated, but the status is not.